config-default.js

```diff
@@ -50,7 +50,10 @@ module.exports = {
 
     // the shell to run when spawning a new session (i.e. /usr/local/bin/fish)
     // if left empty, your system's login shell will be used by default
-    shell: ''
+    shell: '',
+
+    // enable audible bell
+    audibleBell: true
```

Comment:
Sorry for changing the requirements on you last minute. Do you think we should allow `false` *or* a full path to a custom sound? That way plugins can introduce sounds :D